api_tests/share/test_share_preprint.py

```diff
@@ -121,6 +121,7 @@ def test_preprint_contributor_changes_updates_preprints_share(self, mock_share_r
         with expect_preprint_ingest_request(mock_share_responses, preprint):
             preprint.remove_contributor(contributor=user2, auth=auth)
 
+    @pytest.mark.skip('Synchronous retries not supported if celery >=5.0')
```

Comment:
This solution was derived from conversation with MattF and several tests were fixed in [ENG-5526](https://openscience.atlassian.net/browse/ENG-5526), but it seems that one such test still slipped through. To be blunt, I'd also prefer to remove this test, but it may be good case for integration tests if they appear. I've added you to slack discussion on this topic